### PR TITLE
Fixed typo

### PR DESCRIPTION
**Pull request by AnilSeervi**

### CODE REVIEW

**Comment by crowlKats** [comment]
duplicate of #5486

File: std/fs/empty_dir.ts [hunk 1]
```diff
@@ -37,13 +37,13 @@ export async function emptyDir(dir: string): Promise<void> {
...
  * The directory itself is not deleted.
- * Requires the `--allow-read` and `--alow-write` flag.
+ * Requires the `--allow-read` and `--allow-write` flag.
  */
 export function emptyDirSync(dir: string): void {
   try {
     const items = [...readDirSync(dir)];
 
-    // if directory already exist. then remove it's child item.
+    // if directory already exist, then remove it's child item.
```
Comment (piscisaureus) [inline std/fs/empty_dir.ts:46]: After this change, this sentence still contains grammar/spelling errors.

**Comment by piscisaureus** [comment]
If you want to finish the job, feel free to add more commits to #5496,

**Comment by piscisaureus** [comment]
#5527

**Pull request closed without merging**